thoth/adviser/boots/thoth_search.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python3
+# thoth-adviser
+# Copyright(C) 2021 Fridolin Pokorny
+#
+# This program is free software: you can redistribute it and / or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""A boot to provide a link to Thoth search showing results in a UI."""
+
+import logging
+import os
+from typing import Any
+from typing import Dict
+from typing import Generator
+from typing import TYPE_CHECKING
+
+import attr
+
+from ..boot import Boot
+
+if TYPE_CHECKING:
+    from ..pipeline_builder import PipelineBuilderContext
+
+_LOGGER = logging.getLogger(__name__)
+
+_THOTH_SEARCH_URL = os.getenv("THOTH_SEARCH_ADVISER_URL", "https://thoth-station.ninja/search/{document_id}/summary")
```

Comment:
The url was changed due to gh-pages only allowing the url to be the repo name ie `thoth-search`

```
_THOTH_SEARCH_URL = os.getenv("THOTH_SEARCH_ADVISER_URL", "https://thoth-station.ninja/search/{document_id}/summary")
```

Reply:
Adjusted. PTAL. Thanks! 👍🏻

Reply:
> LGTM

Thanks! 👍🏻